supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionBuilder.scala

```diff
@@ -64,6 +64,23 @@ object ActionBuilder {
   def withImage(image: String) = new Action()
     .setImageUri(image)
 
+  /**
+    * The Docker image to report runtime metrics to Stackdriver Monitoring API.
+    * It is currently hard-coded, with the assumption that if folks use PAPI v2,
+    * they would want to get monitoring over a Google service as well.
+    *
+    * However, it could be made into a configuration option overridable
+    * at the workflow and/or Cromwell config levels, at the cost
+    * of additional complexity in configuration.
+    */
+  val monitoringImage = "quay.io/broadinstitute/cromwell-monitor"
```

Comment:
Perhaps this should be in GCR if it's going to be pulled frequently from GCE VMs.

Reply:
Yeah, that'd make sense!

Reply:
Although come to think of it, we won't have to pay if a GCR bucket is multi-regional and accessed through PAPI only.